CHANGELOG.rst

```diff
@@ -26,6 +37,9 @@ Change Log
 0.0.8
 =====
 
+
+0.0.8
+=====
```

Comment:
I think this wrongly-duplicated text must have been an edit-o.

```suggestion
```

Reply:
Done